x-pack/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -87,32 +94,41 @@ export const SessionView = ({ sessionEntityId, height, jumpToEvent }: SessionVie
 
   return (
     <>
-      <EuiFlexGroup>
-        <EuiFlexItem
-          data-test-subj="sessionView:sessionViewProcessEventsSearch"
-          css={{ position: 'relative' }}
-        >
-          <SessionViewSearchBar
-            searchQuery={searchQuery}
-            setSearchQuery={setSearchQuery}
-            onProcessSelected={onProcessSelected}
-            searchResults={searchResults}
-          />
-        </EuiFlexItem>
-        <EuiFlexItem grow={false}>
-          <EuiButton
-            onClick={toggleDetailPanel}
-            iconType="list"
-            fill
-            data-test-subj="sessionViewDetailPanelToggle"
+      <EuiPanel color={'subdued'}>
+        <EuiFlexGroup>
+          <EuiFlexItem
+            data-test-subj="sessionView:sessionViewProcessEventsSearch"
+            css={styles.searchBar}
           >
-            <FormattedMessage
-              id="xpack.sessionView.buttonOpenDetailPanel"
-              defaultMessage="Detail panel"
+            <SessionViewSearchBar
+              searchQuery={searchQuery}
+              setSearchQuery={setSearchQuery}
+              onProcessSelected={onProcessSelected}
+              searchResults={searchResults}
+            />
+          </EuiFlexItem>
+
+          <EuiFlexItem grow={false} css={styles.buttonsEyeDetail}>
+            <SessionViewDisplayOptions
+              optionsStates={optionsStates}
+              onChange={handleOptionChange}
             />
-          </EuiButton>
-        </EuiFlexItem>
-      </EuiFlexGroup>
+          </EuiFlexItem>
+
+          <EuiFlexItem grow={false} css={styles.buttonsEyeDetail}>
+            <EuiButton
+              onClick={toggleDetailPanel}
+              iconType="list"
+              data-test-subj="sessionViewDetailPanelToggle"
```

Comment:
need to add a `sessionView` prefix for the `data-test-subj` (e.g. `data-test-subj="sessionView:sessionViewProcessEventsSearch"` above)